src/_pytest/assertion/rewrite.py

```diff
@@ -102,7 +104,7 @@ def create_module(self, spec):
         return None  # default behaviour is fine
 
     def exec_module(self, module):
-        fn = module.__spec__.origin
+        fn = Path(module.__spec__.origin)
```

Comment:
lol, in my last patch I moved this away from pathlib due to needing `fspath` hax -- this is fine though I guess 🤷‍♂